addons/s3/utils.py

```diff
@@ -143,3 +149,35 @@ def get_bucket_prefixes(access_key, secret_key, prefix, bucket_name):
             )
 
     return folders
+
+
+def update_folder_names():
```

Comment:
Minor: `utils` are usually used in application code, but these additions are exclusively for the command. Would it make more sense to just put them in there?